lib/senders/email.js

```diff
@@ -578,7 +583,8 @@ module.exports = function (log, config, oauthdb) {
         location: this._constructLocationString(message),
         oneClickLink: links.oneClickLink,
         privacyUrl: links.privacyUrl,
-        sync: message.service,
+        serviceName: serviceName,
+        sync: message.service === 'sync',
```

Comment:
I don't see any usage of `sync` in the email template, is it needed?

Reply:
It's used in the built template [here](https://github.com/mozilla/fxa-auth-server/blob/ae2b64198443c9f220f88f0aa18317c367565e49/lib/senders/templates/verify.html#L14).